features/FEATURE_BLE/source/generic/GenericGattClient.tpp

```diff
@@ -966,6 +966,7 @@ GenericGattClient<TPalGattClient, SigningMonitorEventHandler>::GenericGattClient
 	_pal_client->when_transaction_timeout(
 		mbed::callback(this, &GenericGattClient::on_transaction_timeout)
 	);
+	_pal_client->set_event_handler(this);
```

Comment:
does this mean the event handler wasn't set before?

Reply:
Yes 😬